src/arno_evap.c

```diff
@@ -85,6 +85,9 @@ double arno_evap(layer_data_struct *layer,
   double max_infil;
   double Evap;
   double tmpsum;
+  /* best-fitting value of architectural resistance, determined from
+     evaluation of model performance at eddy covariane tower sites */
+  double rarc_soil = 100.0; // [s/m]
```

Comment:
I still don't love this but if @bartnijssen is okay with it, we can keep it for now.  Maybe we should `#define` this in `vicNl_def.h`?

When we port these changes to VIC.5, `rarc_soil` needs to go into the `parameters_struct` or into the `soil_con_struct`.